src/docs/product/data-management-settings/event-grouping/index.mdx

```diff
@@ -57,4 +62,22 @@ If the stack trace is not available, but exception information is, then the grou
 
 Grouping falls back to messages if the stack trace, `type`, and `value` are not available. When this happens, the grouping algorithm will try to use the message without any parameters. If that is not available, the grouping algorithm will use the full message attribute.
 
-For more details, see [Grouping Issues Best Practices](/product/sentry-basics/guides/grouping-and-fingerprints/).
+## Similar Issues Not Grouping
+
+Have you ever had a set of similar-looking issues like this?
+![Issues Dashboard](issues-dashboard.png)
+
+If a set of issues looks similar but they aren't being automatically grouped, there is something that still differs: the stack trace, and therefore the fingerprint. Let's compare two similar looking issues side-by-side:
+
+![Issues Dashboard](issue-stacktraces-comparison.png)
+
+Notice the only difference is one had function `testTypeIssue15` and the other had `testTypeIssue14`. This means the stack traces are not identical.
+
+Sometimes two stack traces are the same function execution path but differ by one frame. This can be due to several things like middleware you've configured, node_modules, the framework itself, or library imports. To have greater control over which stack frames are included or excluded, you can update your grouping algorithm.
+
+There are four different ways in in [sentry.io](https://sentry.io) to update how events group into issues:
+
+- [Fingerprint Rules](/product/data-management-settings/event-grouping/fingerprint-rules/): Sets a new fingerprint for incoming events based on matchers. This doesn't affect already existing issues.
+- [Stack Trace Rules](/product/data-management-settings/event-grouping/stack-trace-rules/): Sets rules for how incoming events should be treated based on matchers. This doesn't affect already existing issues.
+- [Merging Issues](/product/data-management-settings/event-grouping/merging-issues/): Merges together similar issues that have already been created. No settings or configuration changes are required for this.
+- [Grouping Breakdown](/product/data-management-settings/event-grouping/grouping-breakdown/): Creates coarser issue groups by considering the most relevant part of the stack trace.
```

Comment:
this section seems a bit repetitive since we have the same 4 points at the top of the page too. maybe it's a better intro for the page than "all events have a fingerprint"

Reply:
Yeah I was worried about the repetition, but wasn't sure of exactly what to do since the set of links at the top is slightly different from the set of links at the bottom. But without the links, this section wouldn't offer a solution for how to fix the problem. And with regard to the intro, do you mean just the phrase "all events have a fingerprint" or more of the intro? Maybe could you use the suggest feature to show me what you would propose instead?

Reply:
I would update the list on the top to include:

* merging issues
* fingerprint rules
* stack trace rules
* sdk fingerprints
* grouping breakdown

(that order goes from easy to hard/advanced feature)

..then delete this entire section

I believe the links are the same though, i.e. Fingerprint Rules at the top and the bottom point to the same thing.